src/main/java/com/greenfoxacademy/responses/BadRequestResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.greenfoxacademy.responses;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+/**
+ * Created by BSoptei on 2/1/2017.
+ */
+public class BadRequestResponse extends CustomServerResponse {
+
+    @Override
+    public ResponseEntity generateResponse() {
+        return new ResponseEntity<>("", createResponseHeaders(), HttpStatus.BAD_REQUEST);
+    }
+}
```

Comment:
@bsoptei why different classes for creating a ResponseEntity?
the ResponseEntity should be created in the controller, just the first parameters should be defined as different response objects: eg. `ContactListResponse`, `EmptyResponse`, `SingleContactResponse`, etc...
These should be simple classes that can be serialized by the default serializer.